### PR TITLE
fix: remove console log

### PR DESCRIPTION
**Pull request by suyashpatil78**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=OHQTDPO)
### Description
copilot:summary

copilot:poem

### Walkthrough
copilot:walkthrough

## Clickup
app.clickup.com

## Code Coverage
Please add code coverage here

## UI Preview
Please add screenshots for UI changes

### CODE REVIEW

File: src/app/fyle/my-expenses/my-expenses.page.ts [hunk 1]
```diff
@@ -741,7 +741,6 @@ export class MyExpensesPage implements OnInit {
   }
 
   onPageClick(): void {
-    console.log('onPageClick');
```
Comment (OmkarJ13) [inline src/app/fyle/my-expenses/my-expenses.page.ts:744]: https://eslint.org/docs/latest/rules/no-console
We should add this rule for future cases

**Review by OmkarJ13 — APPROVED** [review]

File: src/app/fyle/my-expenses/my-expenses.page.ts (hunk 1 above)
Comment (suyashpatil78) [reply src/app/fyle/my-expenses/my-expenses.page.ts:744]: haa soch rha tha me 🤣 
but in index.html there are console.errors added will that cause problem

Comment (OmkarJ13) [reply src/app/fyle/my-expenses/my-expenses.page.ts:744]: We can disable for that specific case na

Comment (OmkarJ13) [reply src/app/fyle/my-expenses/my-expenses.page.ts:744]: // eslint-disable-next-line no-console

Comment (suyashpatil78) [reply src/app/fyle/my-expenses/my-expenses.page.ts:744]: thike 👍

**Pull request merged**